pkgs/applications/networking/instant-messengers/jami/client.nix

```diff
@@ -31,14 +31,6 @@ stdenv.mkDerivation {
 
   sourceRoot = "source/client-qt";
 
-  patches = [
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Done.